### PR TITLE
volumes section with updated spec

### PR DESCRIPTION
**Pull request by surajssd**
volumes now are defined in service level and root level, root level named volumes has size and mode.

### CODE REVIEW

File: opencompose-specification.asc [hunk 1]
```diff
@@ -143,28 +143,27 @@ ports: tcp::3000
...
-      server: 10.10.1.1
-      path: "/"
+volumes:
+  dataVolume:
+    size: "1Gi"
+    mode: ReadWriteMany
+  staticPages:
+    size: "5Gi"
+    mode: ReadOnlyMany
 ----
```
Comment (pradeepto) [inline opencompose-specification.asc:167]: @surajssd Great stuff! Would be nice to have a complete example that shows how it all holds together would be perfect. And a line about advantages of named volumes would be nice to have as well.

File: opencompose-specification.asc [hunk 2]
```diff
@@ -143,28 +143,54 @@ ports: tcp::3000
 
 
 #### volumes
-A list of paths or named volumes which will be mounted and can be shared by resources.
+Declare volumes used by this container service. The declaration can be done in two ways, as named volume and mount path inside container `VOLUME_NAME:MOUNT_PATH` <1>. The advantages of using named volume is that volume data remains persistent and can be shared with other services.
```
Comment (surajssd) [inline opencompose-specification.asc:146]: Added a line that talks about advantages

File: opencompose-specification.asc [hunk 3]
```diff
@@ -143,28 +143,54 @@ ports: tcp::3000
...
+  dataVolume:
+    size: "1Gi"
+    mode: ReadWriteMany
+  staticPages:
+    size: "5Gi"
+    mode: ReadOnlyMany
+----
+
+Take a look at example below. Here in example see that volume `dataVolume` is being reused by two services. And extra information of these named volumes is defined in root level `volumes` directive.
+
```
Comment (surajssd) [inline opencompose-specification.asc:170]: And example below which has all essential directives with volumes defined.

**Comment by pradeepto** [comment]
LGTM. Thanks @surajssd

**Comment by surajssd** [comment]
@pradeepto thanks, merging!

**Pull request merged**